src/editor/graph/graph_edit.cpp

```diff
@@ -1833,8 +1782,8 @@ void OrchestratorGraphEdit::_on_paste_nodes_request()
 
 void OrchestratorGraphEdit::_on_script_changed()
 {
-    _base_type_button->set_button_icon(SceneUtils::get_editor_icon(_script->get_base_type()));
-    _base_type_button->set_text(vformat("Base Type: %s", _script->get_base_type()));
+    _base_type_button->set_button_icon(SceneUtils::get_editor_icon(_script_graph->get_orchestration()->get_base_type()));
+    _base_type_button->set_text(vformat("Base Type: %s", _script_graph->get_orchestration()->get_base_type()));
```

Comment:
We make a number of calls in this class to `_script_graph->get_orchestration()`, hide this behind a method.